EIPS/eip-draft_multiple_contenthash_records.md

```diff
@@ -0,0 +1,79 @@
+---
+eip: <to be assigned>
+title: Multiple contenthash records for ENS
+author: Filip Štamcar (@filips123)
+discussions-to: https://github.com/ethereum/EIPs/issues/2393
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-18
+requires: 1577
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+This EIP introduces support for multiple `contenthash` records for ENS. It allows hosting a website on multiple distributed systems and accessing it from the same ENS domain. It is an extension of the EIP 1577.
```

Comment:
Simple summary should be thought of like a subject line, GitHub commit message or PR title, forum thread sub-heading, etc.
```suggestion
ENS support for multiple `contenthash` records on a single ENS name.
```